ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml

```diff
@@ -74,6 +74,18 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-network-policy
   timeout: 8h0m0s
+- as: ocp-qe-perfscale-ci-tests-net-perf-gcp
```

Comment:
@krishvoor Can we add node count in the test name for consistency? Will also make it easy for folks to know.

Reply:
You will need to rerun `make jobs`

Reply:
Updated, ptal.